.github/workflows/release-process.yml

```diff
@@ -141,3 +186,81 @@ jobs:
           ${{steps.get-checksums-from-draft-release.outputs.result}}
           EOF
           )
+
+      - name: Updating release notes
+        run: |
+          RELEASE_TAG=$(git describe --tags --abbrev=0)
```

Comment:
Should we try adding html tags to make this section collapsible?

Reply:
collapsible section added.